common/cpp/react/renderer/components/rnscreens/RNSScreenState.h

```diff
@@ -32,6 +32,7 @@ class RNSScreenState final {
   const Size frameSize{};
 
 #ifdef ANDROID
+   __attribute__((visibility("default")))
   folly::dynamic getDynamic() const;
```

Comment:
For some reason this symbol was not visible during linkage of final app's SO as it was not exported from `libreact_codegen_rnscreens.so` library (when investigated with `nm` tool it was marked as a local symbol). Adding this attribute resolves the issue.

Reply:
Note that this explicit GNU compiler attribute was changed in [2cfe611](https://github.com/software-mansion/react-native-screens/pull/1585/commits/2cfe611a7fc27467286f2733ea1f3a87d1958dcf) to usage os `JSI_EXPORT` macro.
Also see this comment: https://github.com/software-mansion/react-native-screens/pull/1585#discussion_r972940173